packages/twenty-front/src/modules/object-record/record-index/components/RecordIndexPageTableAddButtonNoGroup.tsx

```diff
@@ -0,0 +1,21 @@
+import { useRecordIndexContextOrThrow } from '@/object-record/record-index/contexts/RecordIndexContext';
+import { useCreateNewTableRecord } from '@/object-record/record-table/hooks/useCreateNewTableRecords';
+import { PageAddButton } from '@/ui/layout/page/components/PageAddButton';
+import { PageHotkeysEffect } from '@/ui/layout/page/components/PageHotkeysEffect';
+
+export const RecordIndexPageTableAddButtonNoGroup = () => {
```

Comment:
should this be in record table or record index ? does it work for Board too?

Reply:
Actually it doesn't work for RecordBoard for now, I put a `TODO` because to make it work we need to merge some states.
I put the file in this directory because the Kanban one is already there, I think we should move them during the refactor.